main.go

```diff
@@ -453,7 +453,8 @@ func oldInstallExists() bool {
 	if binIsOld {
 		return false
 	}
-	return oldAgentPath.Exist()
+	oldFiles, _ := oldAgentPath.ReadDir()
+	return oldFiles.Len() != 0
```

Comment:
I'd check for the existence of the directory: `~/Applications/ArduinoCreateAgent/ArduinoCreateAgent.app`